services/blockchain-service/blockchain-service-base.js

```diff
@@ -216,7 +225,9 @@ class BlockchainServiceBase {
         return this[blockchain.name].contracts[blockchain.hubContract][contractName];
     }
 
-    async createAsset(requestData, blockchain, stepHooks = emptyHooks) {
+    // Knowledge assets operations
+
+    async createAsset(requestData, paranetKaContract, paranetTokenId, blockchain, stepHooks = emptyHooks) {
```

Comment:
We changed signature here, did we update it everywhere it is used?

Reply:
Maybe rewrite it to be 
createAsset(requestData, blockchain, stepHooks = emptyHooks, paranetKaContract = null, paranetTokenId = null)

Reply:
It is updated everywhere, we use it only internally on 2 places